web/src/beta/features/Editor/DataSourceManager/DelimitedText/index.tsx

```diff
@@ -61,14 +63,11 @@ const DelimitedText: React.FC<DataProps> = ({ sceneId, onSubmit, onClose }) => {
           label="Source Type"
           description="Select the type of data source you want to add.">
           <SourceTypeWrapper>
-            <RadioButtonLabel>
-              <RadioButton
-                value="url"
-                checked={sourceType == "url"}
-                handleChange={c => c && setSourceType("url")}
-              />
-              From URL
-            </RadioButtonLabel>
+            <RadioGroup
+              options={[{ label: t("From URL"), keyValue: "url" }]}
+              layout="horizontal"
```

Comment:
If you do the above recommendation, then remove layout="horizontal" here

Reply:
done